src/shared/mkfs-util.c

```diff
@@ -363,24 +363,6 @@ int make_filesystem(
                                 "-b", "4096",
                                 "-T", "default",
                                 node);
-                if (!argv)
-                        return log_oom();
-
-                if (root && strv_extend_strv(&argv, STRV_MAKE("-d", root), false) < 0)
-                        return log_oom();
-
-        } else if (STR_IN_SET(fstype, "ext3", "ext4")) {
-                argv = strv_new(mkfs,
-                                "-q",
-                                "-L", label,
-                                "-U", vol_id,
-                                "-I", "256",
-                                "-O", "has_journal",
```

Comment:
how "default" is this actually? where did you find that it was? i mean, mke2fs has this non-sensical config file mke2fs.conf (which I am sure noone in his right mind every used). but i couldn't find docs on what the defaults aactually are if that doesn't exist.